packages/react-examples/src/react-next/ExtendedPicker/ExtendedPeoplePicker.Controlled.Example.tsx

```diff
@@ -139,8 +139,8 @@ export class ExtendedPeoplePickerControlledExample extends React.Component<{}, I
       copyMenuItemText: 'Copy name',
       editMenuItemText: 'Edit',
       getEditingItemText: this._getEditingItemText,
-      onRenderFloatingPicker: FloatingPeoplePicker,
-      floatingPickerProps: this._floatingPickerProps,
+      onRenderFloatingPicker: FloatingPeoplePicker as any,
```

Comment:
Do you know why this was needed?

Reply:
It is because the import for SelectedItemsList comes from `react-internal` while FloatingPeoplePicker comes from `react-next`.